Islandora-CLAW/islandora
# Bumps composer versions as per Issue 1052

**GitHub Issue**: /Islandora-CLAW/CLAW/issues/1052

# What does this Pull Request do?

Bumps composer versions for migrate and prepopulate plugins.

# What's new?

* Changes
    - prepopulate 2.0 alpha → 2.2
    - migrate_plus 4.0-beta3 → 4.1
    - migrate_tools 4.0-beta3 → 4.1

# How should this be tested?

Simple test:
    - navigate to '/var/www/html/drupal/web' and run `composer require drupal/prepopulate:^2.2 drupal/migrate_plus:^4.1 drupal/migrate_tools:^4.1`
    - Make sure nothing broke

Full test:
    - checkout a fresh claw-playbook
    - update 'islandora/islandora_demo:dev-8.x-1.x' → 'islandora/islandora_demo:dev-issue-1052' in '[inventory/vagrant/group_vars/webserver/drupal.yml](/Islandora-Devops/claw-playbook/blob/master/inventory/vagrant/group_vars/webserver/drupal.yml#L19)'
    - `vagrant up` should complete as usual
    - make sure nothing broke.
    
# Additional Notes:
Related PR to update devel, rdfui, restui, and facets in the claw_playbook: /Islandora-Devops/claw-playbook/pull/97

Test together by applying it to your fresh claw-playbook before updating the islandora_demo composer dependency to use dev-issue-1052.

# Interested parties
@Islandora-CLAW/committers
